src/centrality/hub_authority.c

```diff
@@ -142,7 +142,7 @@ static igraph_error_t igraph_i_kleinberg_weighted(igraph_real_t *to,
  * \brief Kleinerg's hub and authority scores.
  *
  * The hub and authority scores of the vertices are defined as the principal
- * eigenvectors of <code>A*A^T</code> and <code>A^T*A</code>, respectively, 
```

Comment:
Please revert this file to the original state in the PR; it is great that the trailing whitespace was removed, but the file is unrelated to the PR and I'd rather keep things clean. I'll clean up the trailing whitespace in this file in a separate commit.